docs/admin/runai-setup/notifications/notifications.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Notifications
+summary: This article describes the notifications that are available to the Run:ai platform, and how to configure them.
+authors:
+    - Jason Novich
+date: 2024-Jul-4
+---
+
+## Email Notifications
+
+This document provides an overview of Run:ai's Email Notifications feature, designed to keep data scientists informed about their workload statuses.
+
+### Importance of Email Notifications for Data Scientists
+
+Managing numerous data science workloads requires monitoring various stages, including submission, scheduling, initialization, execution, and completion. Additionally, handling suspensions and failures is crucial for ensuring timely workload completion.
+Email Notifications, introduced in Run:ai version 2.18, address this need by sending alerts for critical workload life cycle changes. This empowers data scientists to take necessary actions and prevent delays.
+
+### Setting Up Email Notifications
+
+The system administrator needs to enable and setup email notifications so that users are kept informed about different system statuses.
+
+To enable email notifications:
```

Comment:
This is for self hosted (SMTP configuration). For SaaS, it's just an option to enable/disable